lib/integration_helpers/test_case/employment_collection.rb

```diff
@@ -84,6 +84,8 @@ def transform_row_to_hash(row, payment_hash)
         payment_hash[:tax] = row[3]
       when "national insurance"
         payment_hash[:national_insurance] = row[3]
+      when "prisoner levy"
+        payment_hash[:prisoner_levy] = row[3]
```

Comment:
don't think we need this change - we're not adding prisoner_levy to our spreadsheet tests?